src/main/java/net/masterthought/cucumber/json/Match.java

```diff
@@ -1,13 +1,95 @@
 package net.masterthought.cucumber.json;
 
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.LinkedList;
+import java.util.List;
+
 public class Match {
 
     // Start: attributes from JSON file report
-    private final String location = null;
+    private final JsonElement location = null;
     // End: attributes from JSON file report
 
     public String getLocation() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The method getLocation() has an NPath complexity of 298](https://www.codacy.com/app/damianszczepanik/cucumber-reporting/file/2183564530/issues/source?bid=3059308&fileBranchId=3097149#l17)